inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/PdfAnnotationEditor.java

```diff
@@ -82,20 +89,12 @@ public void renderHead(IHeaderResponse aResponse)
     @Override
     public void render(AjaxRequestTarget aTarget)
     {
-        try {
-            Selection selection = getModelObject().getSelection();
-            File pdfFile = documentService.getSourceDocumentFile(getModelObject().getDocument());
-            String pdftext = PDFExtractor.processFileToString(pdfFile, false);
-            renderPdfAnnoModel(aTarget, pdftext);
-            if (selection.getAnnotation() != null) {
-                aTarget.appendJavaScript("var anno = pdfanno.contentWindow.annoPage.findAnnotationById("
-                    + selection.getAnnotation().toString() + ");"
-                    + "anno && anno.select();");
-            }
-        }
-        catch (IOException e)
-        {
-            handleError("Could not load data", e, aTarget);
+        Selection selection = getModelObject().getSelection();
+        renderPdfAnnoModel(aTarget);
+        if (selection.getAnnotation() != null) {
+            aTarget.appendJavaScript("var anno = pdfanno.contentWindow.annoPage.findAnnotationById("
+                + selection.getAnnotation().toString() + ");"
```

Comment:
calling `toString()` explicitly here is not necessary - happens implicitly as part of the string concatenation